PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue

```diff
@@ -65,88 +62,77 @@
+                <span class="red--text"><strong>* </strong></span>Date of Birth
+              </template>
+            </v-text-field>
           </template>
-
           <v-date-picker
             reactive
-            v-model="householdDate"
+            v-model="birthDate"
             :min="minDateStr"
             :max="maxDateStr"
           ></v-date-picker>
         </v-menu>
       </v-col>
+      <v-col cols="5">
 
-      <v-spacer></v-spacer>
-    </v-row>
-
-    <v-row align="center" justify="left">
-      <v-col class="d-flex" cols="4" sm="4">
         <!-- Gender identity -->
         <v-select
-          :items="genderID"
-		required
+          :items="genderIDOptions"
+          required
           :rules="[v => !!v || 'Gender identity field is required']"
-          v-model="householdGender"
+          v-model="gender"
           prepend-icon="mdi-menu-right">
-			<template #label>
-			<span class="red--text"><strong>* </strong></span>Gender Identity
-			</template>
+          <template #label>
+            <span class="red--text"><strong>* </strong></span>Gender Identity
+          </template>
         ></v-select>
       </v-col>
-
-      <v-spacer></v-spacer>
-    </v-row>
-
-    <v-row align="left" justify="left">
-      <v-col class="d-flex" cols="4" sm="4">
-        <!-- Current Photo -->
-        <v-file-input
-          accept="image/png, image/jpeg, image/bmp"
-          :rules="[(v) => (v ? v.size : 0) < 2097152 || 'Image size should be less than 2 MB!']"
-          placeholder="Upload a recent photo"
-          v-model="householdPatientPhoto"
-          label="Photo"
-          prepend-icon="mdi-camera"
-        ></v-file-input>
-      </v-col>
     </v-row>
+    <v-row align="center" justify="space-between">
+      <v-col cols="5">
 
-    <v-row align="center" justify="left">
-      <v-col class="d-flex" cols="4" sm="4">
         <!-- Race -->
         <v-select
-          v-model="householdRaceSelections"
-          :items="race"
+          v-model="race"
+          :items="raceOptions"
           label="Race (select all that apply)"
           prepend-icon="mdi-menu-right"
           multiple
         ></v-select>
       </v-col>
-      <v-spacer></v-spacer>
-    </v-row>
+      <v-col cols="5">
 
-    <v-row align="center" justify="left">
-      <v-col class="d-flex" cols="4" sm="4">
         <!-- Ethnicity -->
         <v-select
-          v-model="householdEthnicitySelection"
-          :items="ethnicity"
+          v-model="ethnicity"
+          :items="ethnicityOptions"
           label="Ethnicity"
           prepend-icon="mdi-menu-right"
         ></v-select>
       </v-col>
-      <v-spacer></v-spacer>
+    </v-row>
+    <v-row align="left" justify="space-between">
+      <v-col cols="5">
+
+        <!-- Current Photo -->
+        <v-file-input
+          accept="image/png, image/jpeg, image/bmp"
+          :rules="[(v) => (v ? v.size : 0) < 2097152 || 'Image size should be less than 2 MB!']"
```

Comment:
Can you add in a comment clarifying this logic

Reply:
Added comment (same comment to all 3 files)

Reply:
This change was just introduced due to a merge conflict or something?

Reply:
Yes - in Tom's merge that introduced these changes, the "current photo" was above race and ethnicity options.  I had to cut and paste it to the new position.